crates/bevy_dev_tools/src/fps_overlay.rs

```diff
@@ -28,10 +39,23 @@ pub struct FpsOverlayPlugin {
 
 impl Plugin for FpsOverlayPlugin {
     fn build(&self, app: &mut bevy_app::App) {
+        load_internal_asset!(
+            app,
+            FRAME_TIME_GRAPH_SHADER_HANDLE,
+            "frame_time_graph.wgsl",
+            Shader::from_wgsl
+        );
+
         // TODO: Use plugin dependencies, see https://github.com/bevyengine/bevy/issues/69
         if !app.is_plugin_added::<FrameTimeDiagnosticsPlugin>() {
             app.add_plugins(FrameTimeDiagnosticsPlugin);
         }
+
+        if self.config.frame_time_graph_config.enabled {
```

Comment:
The way we have this now, we can't disable and enable this at runtime, can we maybe just change visibility of the shader?